src/posix/platform/netif.cpp

```diff
@@ -847,7 +847,14 @@ static void processTransmit(otInstance *aInstance)
 
     if (error != OT_ERROR_NONE)
     {
-        otLogWarnPlat("[netif] Failed to transmit, error:%s", otThreadErrorToString(error));
+        if (error == OT_ERROR_DROP)
+        {
+            otLogInfoPlat("[netif] Message dropped by Thread", otThreadErrorToString(error));
```

Comment:
When would a message be dropped?

Reply:
Certain ICMPv6 messages (e.g. MLDv2) are expected to be dropped by Thread.
So, should not print WARN logs since it's perfectly fine.